uint64 timestamp
float64 gyro_x
float64 gyro_y
float64 gyro_z
